js/components/scrubberBar.js

```diff
@@ -402,6 +403,10 @@ var ScrubberBar = createReactClass({
       );
     }
 
+    if (!this.props.audioOnly) {
+      scrubberBarClassName += ' oo-scrubber-bar-video';
```

Comment:
I know that this is how the old code was doing it, but could we use the `className` utility here?

Reply:
Sounds good, will do.